docs/maintainers-guide.md

```diff
@@ -1,87 +1,96 @@
-  - [IntelliJ - Signing Commits with GPG Keys](https://www.jetbrains.com/help/idea/set-up-GPG-commit-signing.html)
-  - [IntelliJ Official YouTube - GPG Commit Signing (10:59)](https://youtu.be/RBhz-8fZN9A?t=659)
+-   **Github**
+    -   [Github - Generating a new GPG key](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key)
+    -   [Github - Adding a GPG key to your Github account](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-gpg-key-to-your-github-account)
+    -   [Github - Configuring your Git CLI for GPG commit signing](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key)
+    -   [Github - Signing Commits with the Git CLI](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)
+    -   [Github - Vigilant Mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits)
+        ‼️
+-   **IntelliJ IDEA (if you use IntelliJ to interact with git)**
+    -   [IntelliJ - Signing Commits with GPG Keys](https://www.jetbrains.com/help/idea/set-up-GPG-commit-signing.html)
+    -   [IntelliJ Official YouTube - GPG Commit Signing (10:59)](https://youtu.be/RBhz-8fZN9A?t=659)
 
 ## Development Model
 
-The Git Source Code Management(SCM) system treats branches as one of the fundamental constructs for supporting 
-concurrent development and assisting with merge conflict resolution. Due to the fundamental nature and flexibility 
-offered by Git branches, it is critical to have a consistent branching workflow in order to reduce merge conflicts and 
-support automation.
+The Git Source Code Management(SCM) system treats branches as one of the fundamental constructs for
+supporting concurrent development and assisting with merge conflict resolution. Due to the
+fundamental nature and flexibility offered by Git branches, it is critical to have a consistent
+branching workflow in order to reduce merge conflicts and support automation.
 
-Git commit tags play a critical role in Release Management strategies and automated releases. Tags are affected by
-branches and branch related operations; therefore, it is critical that the branching workflow incorporates a consistent 
-tagging strategy.
+Git commit tags play a critical role in Release Management strategies and automated releases. Tags
+are affected by branches and branch related operations; therefore, it is critical that the branching
+workflow incorporates a consistent tagging strategy.
 
-[GitFlow branching model](https://nvie.com/posts/a-successful-git-branching-model/) is elected pattern for the 
-development life cycle.
+[GitFlow branching model](https://nvie.com/posts/a-successful-git-branching-model/) is elected
+pattern for the development life cycle.
 
 <p>
     <img src="./assets/gitflow-branching-model.png"/>
 </p>
```

Comment:
We should use markdown syntax for the image inclusion. 

```
<p>
    <img src="./assets/gitflow-branching-model.png"/>
</p>
```

becomes:

`![gitflow-branching-model](./assets/gitflow-branching-model.png)`